src/styles/blocks/card.scss

```diff
@@ -0,0 +1,95 @@
+.card {
+  color: $main-color;
+  display: flex;
+  width: 200px;
+  box-sizing: border-box;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 0 0 3px;
```

Comment:
This margin looks weird, you can just center image using margin: 0 auto; instead